indico/modules/rb_new/blueprint.py

```diff
@@ -96,3 +97,10 @@
 _bp.add_url_rule('/api/admin/attributes', 'admin_attributes', admin.RHAttributes, methods=('GET', 'POST'))
 _bp.add_url_rule('/api/admin/attributes/<int:attribute_id>', 'admin_attributes', admin.RHAttributes,
                  methods=('GET', 'DELETE', 'PATCH'))
+
+# Event linking
+_bp.add_url_rule('!/event/<confId>/manage/rooms/', 'booking_list', event.RHEventBookingList)
+_bp.add_url_rule('!/event/<confId>/manage/contributions/other-list', 'other_contributions',
```

Comment:
not sure what happened to my previous comment, but:

- this is not "other" here unlike in the abstracts version it was copied from
- since the RHs have room-specific logic, I think they belong inside `/manage/rooms/...`, e.g. `!/event/<confId>/manage/rooms/linking/contributions`